curriculum/superblock-structure/full-stack.json

```diff
@@ -547,16 +547,7 @@
           "dashedName": "security-and-privacy",
           "blocks": [
             { "dashedName": "review-security-and-privacy" },
-            { "dashedName": "quiz-security-and-privacy" },
-            { "dashedName": "workshop-npm-packages" },
-            { "dashedName": "lab-timestamp-microservice" },
-            { "dashedName": "workshop-node-and-express" },
-            { "dashedName": "lab-request-header-parser-microservice" },
-            { "dashedName": "lab-url-shortener-microservice" },
-            { "dashedName": "lab-exercise-tracker" },
-            { "dashedName": "lab-file-metadata-microservice" },
-            { "dashedName": "quiz-backend-javascript" },
-            { "dashedName": "quiz-tooling-and-deployment" }
+            { "dashedName": "quiz-security-and-privacy" }
```

Comment:
same goes here 

we decided yesterday that this will be removed too.
This will be added in later once we figure out what we want to do with it.
Please remove the associated files for the security and privacy review